package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@interlay/interbtc-api",
-  "version": "1.16.9",
```

Comment:
If it's a breaking change, might be good to bump the minor version.

Reply:
> If it's a breaking change, might be good to bump the minor version.

You got me. :)

I was trying to "save the bump" for the upcoming removal of some APIs.
But yes, I agree. 
I'm thinking I might deprecate the old API (rather than remove it), with a note that it will be gone in the next minor version. The UI team is expecting this change, and hopefully that isn't too short notice for other consumers we're not aware of.

Reply:
Marking as resolved after adding the old method back, marked as deprecated. Therefore, no breaking changes (yet)